Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -430,6 +432,37 @@ bool MediaPlayerPrivateGStreamerBase::handleSyncMessage(GstMessage* message)
                 GST_WARNING("cannot map %s protection data", eventKeySystemId);
                 break;
             }
+#if USE(OPENCDM)
```

Comment:
Just some general comments here because the patch series needs rework,

 - The code structure need refactoring. Use functions.
 - The WebKit coding style has again been violated in many places.
 - I'm sure there's better function in the GStreamer API to use than munging bytes with memcpy and memcmp, @calvaris  will be better able to review that though. There's certainly better ways to do it in modern C++ and WebKit.

Reply:
<<
The code structure need refactoring. Use functions.
We will split it as function
<<
The WebKit coding style has again been violated in many places.
Usually we use Tools/Scripts/check-webkit-style to cross check webkit style issue in our code. but it is not working in our setup. Is there any issue with that ? Have you faced any issue ?
Could you please provide input on this.
<<
I'm sure there's better function in the GStreamer API to use than munging bytes with memcpy and memcmp, @calvaris will be better able to review that though. There's certainly better ways to do it in modern C++ and WebKit.
Agree with your comment. we can use gst_buffer_copy and gst_buffer_memcmp instead of memcpy and memcmp. In webkit many places, still it is using memcpy and memcmp (even in platform/graphics/gstreamer/eme/WebKitClearKeyDecryptorGStreamer.cpp). @calvaris please suggest us the better option as per you thought.